mockito/invocation.py

```diff
@@ -177,14 +178,18 @@ def thenRaise(self, *exceptions):
             self.__then(Raise(exception))
         return self
 
+    def thenAnswer(self, *callables):
+        for callable in callables:
+            self._AnswerSelector__then(ReturnAnswer(self.invocation.mock, callable))
```

Comment:
Yes, your are absolutely correct. 

I pulled the code out from a project that I'd jerry-rigged to work without change the library.